apis/metal3.io/v1alpha1/baremetalhost_types.go

```diff
@@ -278,6 +278,7 @@ type RAIDConfig struct {
 	// The list of logical disks for hardware RAID, if rootDeviceHints isn't used, first volume is root volume.
 	// You can set the value of this field to `[]` to clear all the hardware RAID configurations.
 	// +optional
+	// +nullable
```

Comment:
Should we include something in the docstring which describes that you can set the value to null, and what that means?